framework/include/st_things/st_things.h

```diff
@@ -152,6 +152,17 @@ int st_things_register_request_cb(st_things_get_request_cb get_cb, st_things_set
  */
 int st_things_start(void);
 
+/**
+ * @brief Send Representation of a specific resource through push service to Samsung Connect client.
+ * @param[in]  resource_uri Resource uri for which we want to send push message.
+ * @param[in]  rep Representation for resource uri.
+ * @return @c 1 on success, otherwise return  value
+ * @retval #ST_THINGS_ERROR_NONE Successful
+ * @retval #ST_THINGS_ERROR_INVALID_PARAMETER Invalid parameter
+ * @retval #ST_THINGS_ERROR_OPERATION_FAILED Operation failed
+ */
+int st_things_push_notification_to_cloud();
```

Comment:
This is a violation of coding rule. And that is M (mandatory).

[M11] In function prototypes, include parameter names with their data types and return type.

[Example-Do]
```
extern int function(struct type *ex);
```
[Example-Don't]
```
extern int function(struct type *);
```